src/openrct2/interface/Viewport.cpp

```diff
@@ -710,7 +718,7 @@ namespace OpenRCT2
 
             if (gLegacyScene != LegacyScene::titleSequence)
             {
-                int32_t height = (TileElementHeight({ sprite->x, sprite->y }))-16;
+                int32_t height = (TileElementHeight({ sprite->x, sprite->y })) - 16;
```

Comment:
clang-format is complaining about this line. I wonder why the outer parentheses are used in the first place, though. Does it still complain if we use the following?
```suggestion
                int32_t height = TileElementHeight({ sprite->x, sprite->y }) - 16;
```